EDMC.py

```diff
@@ -1,27 +1,30 @@
 #!/usr/bin/env python3
-#
-# Command-line interface. Requires prior setup through the GUI.
-#
+"""Command-line interface. Requires prior setup through the GUI."""
 
 
 import argparse
 import json
 import locale
-import logging
 import os
 import re
 import sys
 from os.path import getmtime, join
 from time import sleep, time
-from typing import Any, Optional
+from typing import TYPE_CHECKING, Any, Optional
+
+# isort: off
+from EDMCLogging import edmclogger, logger, logging
+if TYPE_CHECKING:
+    from logging import trace, TRACE  # type: ignore # noqa: F401
```

Comment:
add a `# We add this ourselves at runtime` to this, that way in a few weeks we're not wondering how and why